### PR TITLE
aw-8: add navbar and header

### PR DESCRIPTION
**Pull request by canterbery**
## Describe your changes

Added navbar and header with user avatar

image

## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://testhomework4.atlassian.net/browse/AG-129)

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

### CODE REVIEW

File: src/components/custom/header/header.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import { grey } from '@mui/material/colors';
+import { useTheme } from '@mui/material/styles';
+import { Typography } from '@mui/material';
+import { useMockedUser, useSelector, useTranslations } from 'hooks';
+import { AppRoute } from 'enums';
+import { RootState } from 'store';
+import { HEADER, HeaderAvatar } from './lib';
+import { Logo } from './styles';
+
+function Header() {
```
Comment (AKutusenko) [inline src/components/custom/header/header.tsx]: добавь тип возвр. значения

File: src/components/custom/header/lib/header-avatar.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import Avatar from '@mui/material/Avatar';
+import { AvatarShape } from 'assets/illustrations';
+import { useTranslations } from 'hooks';
+
+type Props = {
+  avatar: string;
+};
+
+const HeaderAvatar = ({ avatar }: Props) => {
+  const t = useTranslations('Home.Header');
```
Comment (AKutusenko) [inline src/components/custom/header/lib/header-avatar.tsx:10]: тоже самое

File: src/components/custom/navbar/navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+'use client';
+
+import AppBar from '@mui/material/AppBar';
+import Toolbar from '@mui/material/Toolbar';
+import { grey } from '@mui/material/colors';
+import { NavSectionHorizontal } from 'components/nav-section';
+import { NAV_ITEMS, defaultConfig } from './lib';
+
+const Navbar = () => {
```
Comment (AKutusenko) [inline src/components/custom/navbar/navbar.tsx]: тоже самое

**Review by AKutusenko — APPROVED** [review]

**Comment by lyudad** [comment]
Screenshot 2023-11-29 at 20 22 24

**Pull request merged**